ui/component/userFirstChannel/view.jsx

```diff
@@ -7,16 +9,27 @@ import { INVALID_NAME_ERROR } from 'constants/claim';
 import Card from 'component/common/card';
 import I18nMessage from 'component/i18nMessage';
 import analytics from 'analytics';
+import { sortLanguageMap } from 'util/default-languages';
+import SUPPORTED_LANGUAGES from 'constants/supported_languages';
+import Spaceman from 'component/channelThumbnail/spaceman.png';
```

Comment:
`spaceman.png` -- we probably don't want to serve any more images directly, especially when one already exists in the CDN.
Would `AVATAR_DEFAULT` cover your needs?